src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/JwtConfig.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app
```

Comment:
OK, I'll move it .... but you'll note with interest that its in the `testIntegration` source, rather than `main`, as its used (needed!) by the integration tests
I'm not 100% sure what it actually does yet, other than if its not there the integration tests dont work!
(SLM has a similar, but more complex !!, version of this class; so I can only deduce it is needed for something or other!)